core/domain/LDAStringClassifierTest.py

```diff
@@ -256,37 +257,52 @@ def test_validate(self):
         }
         self.classifier.validate(classifier_data)
 
-        # Verify Type error is raised when string is provided instead of int.
+        # Verify validation error is raised when key is not present.
+        classifier_data.pop('_alpha', None)
+        with self.assertRaisesRegexp(utils.ValidationError, "key"):
+            self.classifier.validate(classifier_data)
+
+        # Verify Validation error is raised when string is provided instead of
+        # int.
         classifier_data['_alpha'] = 'abc'
-        with self.assertRaises(TypeError):
+        with self.assertRaisesRegexp(utils.ValidationError, "float"):
```

Comment:
Sorry, can you add a bit more detail to the message, so that someone reading the test can easily understand it? E.g.

    with self.assertRaisesRegexp(
            utils.ValidationError, 'Expected alpha to be a float')

Also, use single quotes.

Ditto below.